docs/ballerina-by-example/examples/http-forwarded-extension/http-forwarded-extension.bal

```diff
@@ -0,0 +1,31 @@
+import ballerina.net.http;
+
+service<http> proxy {
+
+    @Description {value:"Proxy server forward the inbound request to a backend with config enabled."}
```

Comment:
Instead of saying `config enabled`, can we be more specific as to what config?

Reply:
Ack